pages/content/amp-dev/documentation/courses/beginning-course/our-first-amp-page@pt_BR.md

```diff
@@ -0,0 +1,451 @@
+
+Além disso, no início e no final de cada um desses cursos, forneceremos um modelo Glitch que inclui todo o código que completamos até aquele ponto. Se você perder sua página atual de Glitch ou quiser começar a partir de nossas soluções, pode copiar o código desses exemplos de Glitch ou simplesmente remixar esses exemplos e seguir em frente a partir daí.
+
+Usando a [documentação](../../../documentation/guides-and-tutorials/start/create/basic_markup.md) para o boilerplate AMP e os comentários acima, atualize seu projeto Glitch para que apenas o erro de validação da tag `<img>` persista. Além disso, para nos ajudar a construir o site da Chico’s Cheese Bikes, fornecemos alguns CSS para usar durante os treinamentos. Se você abrir [esta](https://pastebin.com/vNws2bA1) página, a tag `<style amp-custom>` estará lá com os estilos de que você precisa. Você deve copiar esses estilos para o projeto em que está trabalhando.
+
+### Solução
+
+A solução pode ser encontrada nesse <a href="https://glitch.com/~hungry-modem" target="_blank">examplo do Glitch</a>. A parte da página que contém as alterações deve ser semelhante a esta:
+
+[sourcecode:html]
+{% raw %}<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>Chico's Cheese Bicycles</title>
+  <link rel="shortcut icon" href="https://cdn.glitch.com/d7f46a57-0ca4-4cca-ab0f-69068dec6631%2Fcheese-favicon.png?1540228964214">
+  <meta name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link rel="canonical" href="https://hungry-modem.glitch.me/">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    ... styles elided due to length ...
+  </style>
+</head>
+{% endraw %}[/sourcecode]
+
+## A Valid AMP Page...Almost
+
+Se você completou todos os exercícios acima, parabéns! Sua página é quase uma página AMP válida. Se você abrir sua página e o Chrome DevTools (`Command + Option + I` no Mac ou` Control + Shift + I` no Windows / Linux) e selecionar a guia **Console**, deverá ver esta mensagem no console , indicando que a biblioteca AMP foi carregada com sucesso:
```

Comment:
Maybe the smallest nit ever... but I noticed this because I don't speak Portuguese :) Can you put the backquote (**`**) next to "Control", not next to "ou"?

Reply:
No problem! Done!